src/app/campaign/campaign.component.ts

```diff
@@ -57,9 +58,12 @@ export class CampaignComponent implements OnInit, OnDestroy {
       .pipe(
         switchMap((params: ParamMap) => {
           return this.advertiserService.loadAdvertisers().pipe(map(advertisers => ({ params, advertisers })));
+        }),
```

Comment:
This is the bug fix for accounts. If the campaign loaded before accounts, and the reactive form was bound when accounts had not yet loaded, accounts would not be populated. This makes the campaign load dependent on loading accounts first (same as we're doing for advertisers.)